lab-padget/app/entry.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+require('./scss/main.scss');
+
+const path = require('path');
+//const camelcase = require('camelcase');
+const pascalcase = require('pascalcase');
+const angular = require('angular');
+require('@uirouter/angularjs');
+
+const routesApp = angular.module('routesApp', ['ui.router']);
+
+let context = require.context('./config/', true, /\.js$/);
+context.keys().forEach(key => routesApp.config(context(key)));
+
+context = require.context('./view/', true, /\.js$/);
```

Comment:
yay...context...lol. does this still confuse you? we can chat more, if you'd like